build_docs.py

```diff
@@ -98,8 +159,9 @@ def build_one(version, isdev, quick, sphinxbuild, build_root, www_root,
     maketarget = "autobuild-" + ("dev" if isdev else "stable") + ("-html" if quick else "")
     logging.info("Running make %s", maketarget)
     logname = os.path.basename(checkout) + ".log"
-    shell_out("cd Doc; make SPHINXBUILD=%s %s >> %s 2>&1" %
-              (sphinxbuild, maketarget, os.path.join(log_directory, logname)))
+    shell_out("cd Doc; make SPHINXBUILD=%s SPHINXOPTS='%s' %s >> %s 2>&1" %
+              (sphinxbuild, sphinxopts, maketarget,
+               os.path.join(log_directory, logname)))
```

Comment:
@MarkMangoba @haypo is speaking about the output of this crontab: https://github.com/python/psf-salt/blob/master/salt/docs/init.sls#L47 and [/var/log/docsbuild/](https://github.com/python/docsbuild-scripts/blob/master/build_docs.py#L205).